ui/imports/shared/controls/ContactSelector.qml

```diff
@@ -18,6 +18,7 @@ Item {
     id: root
     property var contacts
     property var selectedContact
+    property int currentIndex: -1
```

Comment:
Since you're doing the same thing here, I'm sure there's a good reason for this. Does `currentIndex` have a special meaning in QML land?